apps/build.sh

```diff
@@ -196,26 +196,24 @@ for x in $IMAGES ; do
 			run docker push ${ct_base}:$TAG-$ARCH
 		fi
 
+		run docker manifest create ${ct_base}:${H_BUILD}_$TAG ${ct_base}:$TAG-$ARCH
+		run docker manifest create ${ct_base}:${LATEST} ${ct_base}:$TAG-$ARCH
+
 		var="EXTRA_TAGS_$ARCH"
 		for t in $(eval echo "\$$var") ; do
 			status "Tagging and pushing extra tag defined in docker-build.conf: $t"
```

Comment:
Maybe the comment should be adjusted accordingly?

Reply:
https://github.com/foundriesio/ci-scripts/pull/198/commits/99c11643beccaf59445136efd38d6ecdaf8d09f8